CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 # Changelog
 
-## v[v1.2.1] - 2023-10-11
+## ?? - 2023-10-31
+
+### Added
+
+- `DetectAndNotifyInactiveUsers` detects inactive users and sends a reminder message to them to login again.
+- `CleanupUsersMarkedForDeletion` deletes all user accounts that do not react after a certain time. All user tokens are removed and study-service is notified. Users are informed by message about the deletion of their account.
+
+New environment variables:
+- `NOTIFY_INACTIVE_USERS_AFTER`: time after which inactivity notification will be triggered in seconds.
+- `DELETE_ACCOUNT_AFTER_NOTIFYING_USER`: length of interval between user notification and deletion of user account in seconds if user does not react. 
+
```

Comment:
We also need the study service address as a new variable, right?
What happens if that is not defined?

Reply:
If study service address is not defined, I would block removing inactive users. All other things work as before as study service is only called when removing inactive users by now.